api/organisations/permissions/migrations/0002_auto_20230410_0940.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.18 on 2023-04-10 09:40
```

Comment:
Can we give this migration a name rather than just the auto generated name from django?